src/views/components/calendar/CalendarBottomBar.tsx

```diff
@@ -22,6 +22,8 @@ export default function CalendarBottomBar({ courseCells, setCourse }: CalendarBo
     const asyncCourseCells = courseCells?.filter(block => block.async);
     const displayCourses = asyncCourseCells && asyncCourseCells.length > 0;
 
+    if (!displayCourses) return <div />;
```

Comment:
Return `null` for an empty case, not an empty div or fragment etc. https://stackoverflow.com/a/58408887/12101554

```suggestion
    if (!displayCourses) return null;
```